source/content/integrated-composer.md

```diff
@@ -0,0 +1,150 @@
+---
+title: Integrated Composer
+description: Learn how to deploy a site with Integrated Composer
+tags: [composer, workflow]
+categories: [drupal]
+contributors: [edwardangert]
+searchboost: 150
+reviewed: "2020-07-21"
+---
+
+Integrated Composer lets you deploy your site on Pantheon with one-click updates for both upstream commits and Composer dependencies, while still receiving upstream updates.
+
+Pantheon is testing a release of Drupal with integrated Composer and is opening the feature to users for Limited Availability testing. 
+
+## Get Access to Integrated Composer
+
+Integrated Composer is currently limited to customers with a Pantheon [organization](/organizations). If you are not part of an organization already, contact [Support](/support) to request access and your CSE will add you to a test organization from which you can deploy the upstream. If you are already part of an organization, the feature can be enabled organization-wide.
+
+## Create a Drupal or WordPress Site with Integrated Composer
+
+1. [Fork the Pantheon-maintained repository](/create-custom-upstream#create-and-host-the-repository-remotely):
+
+   - **Drupal**: [https://github.com/pantheon-upstreams/drupal-project](https://github.com/pantheon-upstreams/drupal-project).
+
+   - **WordPress**: [https://github.com/pantheon-upstreams/wordpress-project](https://github.com/pantheon-upstreams/wordpress-project).
+
+1. [Add a new Custom Upstream](/create-custom-upstream#connect-repository-to-pantheon) on the Pantheon dashboard.
+
+1. Create a new Drupal 8 or WordPress site from the upstream to confirm that it’s working.
+
+   - Do not customize the upstream yet.
+
+1. In the Dev environment, click **Visit Development Site** and follow the prompts to complete the CMS installation.
+
+1. [Clone the site locally](/local-development#get-the-code) and run `composer install`.
+
+## Upstream and Site Structure
+
+The upstream has the following directory structure:
+
+```none:title=core/
+core/
+├─ .gitignore
+├─ composer.json
+└─ pantheon.upstream.yml
+├─ README.md
+└─ upstream-config/
+   └─ composer.json
+```
+
+- `.gitignore`: Prevents build artifacts generated by Composer from being committed to the upstream or site code repositories.
+- `composer.json`: The two different `composer.json` files allow customization of individual sites without inherent merge conflicts and enable one-click updates.
+  - Root-level: Site-level customizations.
+  - `upstream-config/composer.json`: Customizations for the upstream
+- `pantheon.upstream.yml`: The `build_step_demo: true` directive in `pantheon.upstream.yml` enables the build step. This name is temporary and will change.
+
+When a site is created, Pantheon runs `composer install`, generates a `composer.lock` file and commits it back to the site’s code repository.
+
+Build artifacts are stored in a Git tag (`pantheon_build_artifacts_master`), not the main branch (`master` for Dev or a Multidev feature branch).
```

Comment:
If we intend to go into the implementation details, there's a caveat here in that the tag name is dependent upon the multidev env, thus its name is: `pantheon_build_artifacts_{BRANCH_NAME}`

Reply:
thanks! I adjusted [L61](https://github.com/pantheon-systems/documentation/pull/5663/commits/90f5dfddf836f76060f647d6f37ae69258e32e13#diff-c609f8474a4e653f4ee707364d61540ac160cd501467d72eec5663504401d3ddR61) to try to explain this